pkgs/os-specific/linux/conky/default.nix

```diff
@@ -166,46 +159,48 @@ stdenv.mkDerivation rec {
     ++ lib.optional wirelessSupport wirelesstools
     ++ lib.optional curlSupport curl
     ++ lib.optional rssSupport libxml2
-    ++ lib.optional weatherXoapSupport libxml2
     ++ lib.optional nvidiaSupport libXNVCtrl
     ++ lib.optional pulseSupport libpulseaudio
     ++ lib.optional journalSupport systemd;
 
-  cmakeFlags =
-    [ ]
-    ++ lib.optional docsSupport "-DMAINTAINER_MODE=ON"
-    ++ lib.optional curlSupport "-DBUILD_CURL=ON"
-    ++ lib.optional (!ibmSupport) "-DBUILD_IBM=OFF"
-    ++ lib.optional imlib2Support "-DBUILD_IMLIB2=ON"
-    ++ lib.optional luaCairoSupport "-DBUILD_LUA_CAIRO=ON"
-    ++ lib.optional luaImlib2Support "-DBUILD_LUA_IMLIB2=ON"
-    ++ lib.optional (!mpdSupport) "-DBUILD_MPD=OFF"
-    ++ lib.optional (!ncursesSupport) "-DBUILD_NCURSES=OFF"
-    ++ lib.optional rssSupport "-DBUILD_RSS=ON"
-    ++ lib.optional (!x11Support) "-DBUILD_X11=OFF"
-    ++ lib.optional waylandSupport "-DBUILD_WAYLAND=ON"
-    ++ lib.optional xdamageSupport "-DBUILD_XDAMAGE=ON"
-    ++ lib.optional doubleBufferSupport "-DBUILD_XDBE=ON"
-    ++ lib.optional weatherMetarSupport "-DBUILD_WEATHER_METAR=ON"
-    ++ lib.optional weatherXoapSupport "-DBUILD_WEATHER_XOAP=ON"
-    ++ lib.optional wirelessSupport "-DBUILD_WLAN=ON"
-    ++ lib.optional nvidiaSupport "-DBUILD_NVIDIA=ON"
-    ++ lib.optional pulseSupport "-DBUILD_PULSEAUDIO=ON"
-    ++ lib.optional journalSupport "-DBUILD_JOURNAL=ON";
-
-  # `make -f src/CMakeFiles/conky.dir/build.make src/CMakeFiles/conky.dir/conky.cc.o`:
-  # src/conky.cc:137:23: fatal error: defconfig.h: No such file or directory
-  enableParallelBuilding = false;
```

Comment:
we still need `enableParallelBuilding = false`. i tried building on a 12 core machine:
- without `enableParallelBuilding = false`: 3 out of 6 builds fail (each with different error messages).
- with `enableParallelBuilding = false`: 0 out 6 builds fail.

Reply:
i just ran another 10 runs and only captured _one_ failure: guess i was pretty (un?)lucky the first time around.

[here's](https://gist.github.com/uninsane/51bbee0e20a061a38f91b2c5407dcbfa) a full nix log for the failure, and then excerpts from two earlier failures that i happened to record. nix hashes will differ because i was forcing rebuilds by just incrementing some unused `env.BUILD_COUNT = "...";` attribute.

Reply:
That doesn't look like a `enableParallelBuilding` issue as such to me; last time I had these kinds of errors in builds (not in conky) during a large nixpkgs bisection my PC was just kind of melting and I had to do a BIOS update to resolve it.

Could you try a large rebuild of something (like libreoffice, firefox or thunderbird maybe) and see if you get similar issues?

Reply:
Mind if I enable the parallel build again then?

Reply:
yeah go ahead & re-enable parallel building

Reply:
Cheers, thanks for reviewing and testing!